### PR TITLE
Nav func

### PR DESCRIPTION
**Pull request by kimmccaskill**
## What functionality does this accomplish?
closes #

**Description:**
- Add tab navigation (switch between screens with bottom tabs)
- Add stack navigation (switch between screens by clicking buttons inside of particular tab)

Screen Shot 2020-04-07 at 12 38 01 PM

## What did you struggle on to complete?
- Had to rework some of the file structure

### Test Coverage Percentage: x%
- [x] No Tests have been changed
- [ ] New Tests have been implemented
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):

## Checklist:
- [ ] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

## Helpful Resources:
* Resource link AND small description of info gathered/learned
[Tab navigation](https://reactnavigation.org/docs/tab-based-navigation)
[Stack navigation](https://reactnavigation.org/docs/navigating)

## Please include a gif of how you feel about this branch:
dead

### CODE REVIEW

File: src/screens/tabs/PrevTab.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+
+const PrevTab = () =>
+  <View >
+    <Text>Tab 1</Text>
```
Comment (kimmccaskill) [inline src/screens/tabs/PrevTab.js:6]: This is a placeholder tab

File: src/screens/FormScreen.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react'
-import { StyleSheet, View, Modal, Text, Picker, Button } from 'react-native'
+import { StyleSheet, View, Text, Picker, Button } from 'react-native'
 
-const Form = ({visible, setShowForm, setShowResult}) => {
+const FormScreen = ({ navigation }) => {
```
Comment (CHaiz15) [inline src/screens/FormScreen.js:4]: Much cleaner! 👏

File: src/screens/StartScreen.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          onPress={() => navigation.navigate('Form')}
+        />
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    // paddingTop: 50,
```
Comment (CHaiz15) [inline src/screens/StartScreen.js:20]: 👀👀

Comment (kimmccaskill) [reply src/screens/StartScreen.js:20]: I forgot to delete this comment but the padding was commented out due to stack navigation having its own styling for the header which resulted in the padding being unnecessary

**Pull request merged**